tests/test_events.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from celery import Celery
+from flask import Flask
+from flask.json import jsonify, dumps
+from flask.ext.notifications import NotificationService, Event
+
+
+class EventsTest(unittest.TestCase):
+    def setUp(self):
+        self.app = Flask(__name__)
+
+        self.config = {
```

Comment:
Sorry, what do you mean by console? I was thinking about just importing config from the main app.